templates/arm-aks-template.json

```diff
@@ -178,7 +178,8 @@
           "networkPlugin": "azure",
           "serviceCidr": "[parameters('aksServiceCidr')]",
           "dnsServiceIP": "[parameters('aksDnsServiceIp')]",
-          "dockerBridgeCidr": "172.17.0.1/16"
+          "dockerBridgeCidr": "172.17.0.1/16",
+          "loadBalancerSku": "standard"
```

Comment:
do you need to provision the standard public ip as well?

Reply:
ideally that would be not put in the cluster resource group so that we keep the same IP address on rebuilds that way we would have a stable egress ip